queries.sql

```diff
@@ -118,3 +118,47 @@ SELECT species AS type_of_animal, AVG(escape_attempts) AS average_of_escape_atte
 FROM animals
 WHERE date_of_birth BETWEEN '1990-01-01' AND '2000-12-31'
 GROUP BY type_of_animal;
+
```

Comment:
You are required to add the following data to the data.sql file which is  currently unavailable 
- for the owner's table data
```
 INSERT INTO owners (full_name, age)
 VALUES ('Sam Smith', 34);

 INSERT INTO owners (full_name, age)
 VALUES ('Jennifer Orwell', 19);

 INSERT INTO owners (full_name, age)
 VALUES ('Bob', 45);

 INSERT INTO owners (full_name, age)
 VALUES ('Melody Pond', 77);

 INSERT INTO owners (full_name, age)
 VALUES ('Dean Winchester', 14);

 INSERT INTO owners (full_name, age)
 VALUES ('Jodie Whittaker', 38);
```
- for the species table  data
```
 INSERT INTO species (name)
 VALUES ('Pokemon');

 INSERT INTO species (name)
 VALUES ('Digimon');
```

- `animals` table should be modified to meet these requirements
     - if an animal's name ends with `mon` it should have `Digimon` as a `species_id`, all other animals should have `Pokemon` instead
     - all animals got assigned owners

Reply:
Thank you, I did not notice that my changes were not committed. @alexander16108